cfg/rke2-cis-1.24/master.yaml

```diff
@@ -979,7 +981,7 @@ groups:
           Edit the Controller Manager pod specification file $controllermanagerconf
           on the control plane node and set the --feature-gates parameter to include RotateKubeletServerCertificate=true.
           --feature-gates=RotateKubeletServerCertificate=true
-        scored: true
+        scored: false
```

Comment:
I'm still not sure about this. but if you're - we can merge it

my concerns are next:
* the mentioned issue - https://github.com/rancher/dashboard/issues/4485, added a new feature - Rotate Certificates, if i understood correctly. (last updated on March,2022)
* CIS from RKE2 tells us to [Ensure that the RotateKubeletServerCertificate argument is set to true](https://docs.rke2.io/security/cis_self_assessment124#136-ensure-that-the-rotatekubeletservercertificate-argument-is-set-to-true-automated) (Last updated on Sep 18, 2024)

but we want to ignore it. 

What did I miss? thanks!

Reply:
These are two separate features, and based on this, As I see it we should not be skipping the check.

@sm171190 The --feature-gates option has been removed. You can verify this at https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/ to see if it helps clarify the issue.

Reply:
@afdesk Based on the reference shared by @ManojShastha https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/ the RotateKubeletServerCertificate feature gate was in GA from v1.19 to v1.21. SInce we're dealing with checks on k8s 1.24 I think we can skip it. Whatsay?